### PR TITLE
Delete `standardtracer` and clean up

### PR DESCRIPTION
**Pull request by tschottdorf**

### CODE REVIEW

**Comment by bensigelman** [comment]
:ship:!

File: ext/tags_test.go [hunk 1]
```diff
@@ -1,20 +1,27 @@
 package ext_test
 
 import (
+	"reflect"
 	"testing"
 
+	"github.com/opentracing/basictracer-go"
+	"github.com/opentracing/basictracer-go/testutils"
```
Comment (bensigelman) [inline ext/tags_test.go]: this is a little awkward from a deps standpoint, but for now, "whatever."

Comment (tschottdorf) [inline ext/tags_test.go]: Should be ok, it's only a test dependency. If it becomes an issue, someone will have to put a trivial tracer impl that supports attributes here.

Comment (yurishkuro) [inline ext/tags_test.go]: why not just use noop tracer? I think the only point of these tests (at least in python) was to get test coverage to 100%. In Go they have additional benefit of validating signatures, but we don't need asserts to read back the values imo.

File: examples/dapperish/dapper.go [hunk 2]
```diff
@@ -1,11 +1,11 @@
 package dapperish
 
 import (
+	"github.com/opentracing/basictracer-go"
```
Comment (yurishkuro) [inline examples/dapperish/dapper.go]: can we move daperish into basic tracer too?  these recursive imports look strange.

File: ext/tags_test.go (hunk 1 above)
Comment (tschottdorf) [inline ext/tags_test.go]: SGTM

**Pull request merged**